python/lsst/ip/isr/isrStatistics.py

```diff
@@ -251,6 +268,11 @@ def run(self, inputExp, ptc=None, overscanResults=None, **kwargs):
                 correction applied (`lsst.afw.image.Image`). This
                 quantity is used to estimate the amplifier read noise
                 empirically.
+        doLegacyCtiStatistics : `bool`, optional
+            Use the older version of measureCti (not recommended).
+            This should be True if and only if this task is called
+            from IsrTask. TODO: Deprecate legacy CTI + CTI correction
+            from IsrTask.
```

Comment:
Can you file this deprecate ticket and link it here?  It would need an RFC to be done, but we can file the plan now.